includes/footer.php

```diff
@@ -1,3 +1,32 @@
 <!-- START FOOTER -->
+    <footer>
+        <div class="row">
+            <a href="/" class="one column"><i class="far fa-envelope"></i></a>
+            <a href="/" class="one column"><i class="fas fa-mobile-alt"></i></a>
+            <a href="/" class="one column"><i class="fas fa-question-circle"></i></a>
+            <a href="/" class="one column"><i class="fas fa-user-circle"></i></a>
+
+            <p style="background-color: #2E9E47" class="four columns">Treat Street</p><!--replace with image-->
+
+            <a href="/" class="one column"><i class="fab fa-facebook"></i></a>
+            <a href="/" class="one column"><i class="fab fa-twitter"></i></a>
+            <a href="/" class="one column"><i class="fab fa-instagram"></i></a>
+            <a href="/" class="one column"><i class="fas fa-map-marker-alt"></i></a>
+
+        </div>
+
+        <hr>
```

Comment:
What is the purpose of this <hr> ? If it's just to add a border then we probably want to remove it and replace with CSS later. Take a look at the documentation for <hr>: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr

They describe the purpose of a horizontal rule (<hr>) as: "a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section."

Let me know what you think 😄

Reply:
The hr tag has been removed. Even if it was meant to be a thematic break, I think that is something that should be determined later.